Tests/SwiftLintFrameworkTests/GlobTests.swift

```diff
@@ -1,4 +1,3 @@
-import Foundation
```

Comment:
If exist, CI failed

Reply:
Yes, this is because we run the [`unused_import` analyzer rule](https://realm.github.io/SwiftLint/unused_import.html).